daemon/src/taker.rs

```diff
@@ -200,72 +179,43 @@ async fn main() -> Result<()> {
                 housekeeping::rebroadcast_transactions(&mut conn, &wallet)
                     .await
                     .unwrap();
```

Comment:
I think with https://github.com/comit-network/hermes/pull/381, there is actually no more reason why we cannot boot up the actor system outside of rocket.

Doing that would allow us for easier error handling because we can just use `?` instead of `unwrap`ing in here.

Best to be done in a follow-up though!

Reply:
happy to do it as a follow-up :)